oas_docs/overlays/kibana.overlays.yaml

```diff
@@ -102,10 +102,29 @@ actions:
         externalDocs:
           description: Synthetic monitoring
           url: https://www.elastic.co/docs/solutions/observability/synthetics
+  - target: '$.tags[?(@.name=="Fleet remote synced integrations")]'
+    description: Change tag description and add external docs link
+    update:
+      description: >
+        Use the Fleet remote synced integrations API to check the status of the automatic integrations synchronization on a remote cluster:
+
+        * Use the `/api/fleet/remote_synced_integrations/{outputId}/remote_status` endpoint on the management cluster to query the synchronization status of the integrations installed on the remote cluster by the ID of the configured remote Elasticsearch output.
+        
+        * Use the `/api/fleet/remote_synced_integrations/status` endpoint on the remote cluster to query the synchronization status of the installed integrations.
+      externalDocs:
+        description: Automatic integrations synchronization documentation
+        url: https://www.elastic.co/docs/reference/fleet/automatic-integrations-synchronization
 # Remove extra tags from operations
   - target: "$.paths[*][*].tags[1:]"
     description: Remove all but first tag from operations
     remove: true
+# Add availability for operations
```

Comment:
> # Add availability for operations

Rather than adding these via overlays, they should be added to the source files. For example, per my suggestion in https://github.com/elastic/kibana/pull/233482#discussion_r2311683006